citations-tool/tool/src/webapp/js/jquery.googlebooks.thumbnails.js

```diff
@@ -0,0 +1,27 @@
+/**
+ * JQuery Plugin to get thumbnail images from Google Books
+ *
+ *  Given an img (or list of img's), which is representing a thumbnail for a media type,
+ *  that has a class of 'citationThumbnail' and an id containing its ISBN,
+ *  this plugin populates the src of that img with a small thumbnail from the Google Books API.
+ */
+(function ($) {
+
+    var GOOGLE_BOOKS_SEARCH_API = 'https://www.googleapis.com/books/v1/volumes?';
+
+    $.fn.setGoogleBooksThumbnailURL = function(element) {
+        if (element.id!=''){
+            var googleBooksURL = GOOGLE_BOOKS_SEARCH_API + 'q=isbn:' + element.id;
+            $.getJSON(googleBooksURL, function(data){
+                var imgURL;
+                try {
+                    imgURL = data.items[0].volumeInfo.imageLinks.smallThumbnail;
+                    element.src = imgURL;
+                }
+                catch(err) {
+                    console.log ('There was a problem getting the small thumbnail using this URL:' + googleBooksURL );
```

Comment:
You should normally test before using console.log so that if it's not defined you don't get another error: `window.console && console.log('foo');` in this case it doesn't matter as it's the error handler but it nice todo it so that if the code gets refactored the refactored code doesn't have any problems.